### PR TITLE
fix(1755): Add baseBranch column to events migration

### PR DESCRIPTION
**Pull request by kumada626**
## Context

I added `baseBranch` to events schema in #357 .
Therefore, I added `baseBranch` column to events init migration.

## Objective

Add `baseBranch` column to events init migration.

## References

#357 

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tk3fftk** [comment]
related discussion: https://github.com/screwdriver-cd/data-schema/pull/356#issuecomment-537590983

**Comment by kumada626** [comment]
@jithin1987 I created a new migration file.

File: migrations/20191007-add_baseBranch_column_to_events.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}events`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.createTable(table, {
```
Comment (tk3fftk) [inline migrations/20191007-add_baseBranch_column_to_events.js]: `createTable` -> `addColumn` ?
Refs:
- https://sequelize.org/master/manual/migrations.html
- https://sequelize.org/master/class/lib/query-interface.js~QueryInterface.html#instance-method-addColumn

Comment (kumada626) [reply migrations/20191007-add_baseBranch_column_to_events.js]: That's true. Fixed to use `addColumn`.

**Pull request merged**